proto/destination.proto

```diff
@@ -120,3 +125,73 @@ message ProtocolHint {
 
   message H2 {}
 }
+
+message DestinationProfile {
+  // A list of routes, each with a RequestMatch.  If a request matches
+  // more than one route, the first match wins.
+  repeated Route routes = 1;
+}
+
+message Route {
+  // This route contains requests which match this condition.
+  RequestMatch condition = 1;
+  // A list of response classes for this route.  If a response matches
+  // more than one ResponseClass, the first match wins.  If a response does not
+  // match any ResponseClasses, it is considered to be a successful response.
+  repeated ResponseClass response_classes = 2;
+}
+
+message ResponseClass {
+  // This class contains responses which match this condition.
+  ResponseMatch condition = 1;
+  // If responses in this class should be considered failures.
+  bool is_failure = 2;
```

Comment:
Just noting that bools default to `false`... mostly a concern for if this field is ever removed in the future.